### DISCUSSION TITLE
[mpvnet] 如何注册成默播放器？

### DISCUSSION THREAD

**Discussion by jolyonaha** [hooke007/mpv_PlayKit#149] [libmpv]
mpv.net_CM没有installer文件夹。纯小白，问下该怎么注册

**Comment by jolyonaha** [comment]
![image](https://user-images.githubusercontent.com/67818339/179009198-b2992ba0-2ce5-44cb-85bb-f4dde90b726d.png)
已解决，我刚刚瞎了

**Reply by ghost** [reply]
我也瞎了
